docs/platforms/apple/common/index.mdx

```diff
@@ -119,6 +119,13 @@ func application(_ application: UIApplication,
 
     return true
 }
+
+// Start continuous profiling when the app becomes active again.
+//
+// This is optional, and you can start and stop the profiler as needed.
+func applicationDidBecomeActive(_ application: UIApplication) {
+    SentrySDK.startProfiler()
+}
```

Comment:
I'm not sure if this doesn't add too much noise. We already mention `SentrySDK.startProfiler()` above and tell the user that they have to manually start and stop the profiler.